src/tv2-common/actions/executeAction.ts

```diff
@@ -1973,4 +1989,19 @@ function executeActionClearGraphics<
 			tags: userData.sendCommands ? [TallyTags.GFX_CLEAR] : [TallyTags.GFX_ALTUD]
 		})
 	)
+
+	markPartAsModifiedByAction(context, 'current')
+}
+
+function markPartAsModifiedByAction(context: IActionExecutionContext, part: 'current' | 'next') {
```

Comment:
Does this have to be called **every** time an action creates/modifies a PartInstance? If yes, perhaps it should be a core thing to set such property, or we should somehow wrap any context methods into our own that would set this property to prevent potential bugs caused by forgetting to do this

Reply:
Yeah any time modifications are made, but this is just for our own (TV 2) logic of tracking whether next has been modified or not

Reply:
I've wrapped the context like so: 632c85bee34d94d15790b221d124220bb5709fd0